flocker/provision/_install.py

```diff
@@ -128,15 +164,39 @@ def task_install_digitalocean_kernel():
     ])
 
 
-def task_upgrade_kernel_centos():
-    return sequence([
-        run_from_args([
-            "yum", "install", "-y", "kernel-devel", "kernel"]),
-        # For dkms and ... ?
-        run_from_args([
-            "yum", "install", "-y", "epel-release"]),
-        run_from_args(['sync']),
-    ])
+def task_install_requirements(distribution):
+    if distribution == 'ubuntu-14.04':
+        return sequence([
+            # Add ZFS repo for recent ZFS versions - XXX no minimum
```

Comment:
Looking into it further, it seems Ubuntu does not have ZFS and Docker available without these commands - i.e. it is not just outdated versions, so I have changed the comments